### PR TITLE
Zipping backup-history folder( assuming backup-history folder exists)

### PR DESCRIPTION
**Pull request by rajashankary**
https://github.com/phillipcheng/log.analysis/commit/6c475b0f8850637fa9117228be3ef2b5742df197

### CODE REVIEW

**Comment by phillipcheng** [comment]
Data-history folder is used for backup.
1.	So I think you do not need to define another property backup-history=/mtccore/backup-history (so that you donot need to change sgsiwf.backup.properties and DynSchemaCmd.java)
2.	You need to backup (zip) all the raw input files (xml files) and intermediate files (csv files) belongs to this batch (wfid) into 1 zip file named as $(wfid).zip. wfid is passed to the command.
a.	All xml files names can be get from dynCfgMap.get(DynSchemaCmd.dynCfg_Key_XML_FILES);
b.	All csv files names are under this convention: 
private String getOutputDataFileName(String tableName, String timeStr){return csvFolder + timeStr + "_" + tableName + ".csv";}
so basically you need to add the csvfiles begin with wfid

3.	Do not use system.out to print logs, please use logger.info
4.	zos.closeEntry(); should be used to close each entry should be put inside of the for loop.

5.	Should we put zos.close in the finally block

Could you please finish those above items and test again?
